tools/ALARAJOYWrapper/alarajoy_QA.py

```diff
@@ -112,99 +109,242 @@ def run_alara(element, libname):
-            'Unit': unit,
-            'Data': data
-        }
-    }
-
-def process_data(
-        data_source,
-        inp_datalib=None,
-        inp_variable=None,
-        inp_unit=None
-        ):
+    if '(' in label:
+        parts = label.split('(')
+        isotope = parts[0].strip()
+        run_lbl = f'({parts[1].strip(')')})'
+    else:
+        isotope = label.strip()
+        run_lbl = ''
+    return isotope, run_lbl
+
+def reformat_isotope(isotope):
     '''
-    Flexibly create a dictionary of subdictionaries containing Pandas
-        DataFrames with associated metadata containing ALARA output data for
-        different variables. Allows for processing of either existing
-        DataFrames, with the requirement that the user provide the relevant
-        data source, evaulated variable, and its relevant units to be packaged
-        into a data dictionary. Alternatively, this function can directly
-        read in an ALARA output file and parse all tables and their metadata
-        internally.
+    Restructure the string describing an isotope to capitalize its first
+        letter and place the atomic number in a superscript for cleaner
+        presentation in legend. Skips "isotope" entries of "total" or "Other".
     Arguments:
-        data_source (dict or pandas.core.frame.DataFrame): ALARA output data.
-            If parsing directly from ALARA output files, data_source must be
-            formatted as a dictionary of the form:
-            data_source = {
-                Data Library 1 : path/to/output/file/for/data/library1,
-                Data Library 2 : path/to/output/file/for/data/library2
-            }
-            If processing a preexisting DataFrame, then data_source is just
-            the DataFrame itself.
-        inp_datalib (str or None, optional): Data source of the selected
-            DataFrame. Required if processing a preexisting DataFrame;
-            irrelevant if parsing directly from ALARA output files.
-            (Defaults to None)
-        inp_variable (str or None, optional): Evaluated variable for the
-            selected DataFrame. Required if processing a preexisting
-            DataFrame; irrelevant if parsing directly from ALARA output files.
-            (Defaults to None)
-        inp_unit (str or None, optional): Appropriate unit for the evaluated
-            variable for the selected DataFrame. Required if processing a
-            preexisting DataFrame; irrelevant if parsing directly from ALARA
-            output files.
-            (Defaults to None)
+        isotope (str): Identifier of the isotope of the form element-A.
+    
+    Returns:
+        isotope (str): Reformatted identifier of the isotope of the form
+            ᴬelement.
+    '''
+
+    if isotope == 'total' or isotope == 'Other':
+        return isotope
+    else:
+        element, A = isotope.split('-')
+        element = element.capitalize()
+        return f'$^{{{A}}}\\mathrm{{{element}}}$'
```

Comment:
This might be simpler and equivalent since `mathrm` is basically the same as non-math
```suggestion
        return f'$^{{{A}}}${element}'
```